src/pages/gateway/command-reference.md

```diff
@@ -82,28 +69,104 @@ Retrieves the current `JSON` mesh file for the specified mesh.
 
 ### Example
 
 ```bash
-aio api-manager:mesh:get meshId1234
+aio api-manager:mesh:get continuing-aqua-bobby-2pUkDmZd
 ```
 
 #### Response
 
 ```terminal
-get meshId1234
-Selecting your organization as: my-organization-name
-Initialized user login and the selected organization
-OrgCode - getMesh: 27E41B3246BEC9B16E398115@MyOrg
-Config : [object Object]
-{"imsOrgId":"27E41B3246BEC9B16E398115@MyOrg","lastUpdated":"1234123412341","meshConfig":{"sources":[{"name":"Commerce","handler":{"graphql":{"endpoint":"https://<your_commerce_site>/graphql/"}}},{"name":"AEM","handler":{"graphql":{"endpoint":"https://<your_AEM_site>/endpoint.json"}}},{"name":"LiveSearch","handler":{"graphql":{"endpoint":"https://<your_commerce_site>/search/graphql","operationHeaders":{"Magento-Store-View-Code":"default","Magento-Website-Code":"base","Magento-Store-Code":"main_website_store","Magento-Environment-Id":"<your_environment_id>","x-api-key":"search_gql","Content-Type":"application/json"},"schemaHeaders":{"Magento-Store-View-Code":"default","Magento-Website-Code":"base","Magento-Store-Code":"main_website_store","Magento-Environment-Id":"<your_environment_id>","x-api-key":"search_gql","Content-Type":"application/json"}}}}]},"meshId":"meshId1234","lastUpdatedBy":{"firstName":"User","lastName":"Name","userEmail":"uname@domain.com","userId":"undefined","displayName":"User%20Name"}}
+Successfully retrieved mesh {
+  "lastUpdated": "2022-06-01T12:12:12.0000",
+  "meshConfig": {
+    "sources": [
+      {
+        "name": "Commerce",
+        "handler": {
+          "graphql": {
+            "endpoint": "https://<your_commerce_site>/graphql/"
+          }
+        }
+      },
+      {
+        "name": "AEM",
+        "handler": {
+          "graphql": {
+            "endpoint": "https://<your_AEM_site>/endpoint.json"
+          }
+        }
+      },
+      {
+        "name": "LiveSearch",
+        "handler": {
+          "graphql": {
+            "endpoint": "https://<your_commerce_site>/search/graphql",
+            "operationHeaders": {
+              "Magento-Store-View-Code": "default",
+              "Magento-Website-Code": "base",
+              "Magento-Store-Code": "main_website_store",
+              "Magento-Environment-Id": "<your_environment_id>",
+              "x-api-key": "search_gql",
+              "Content-Type": "application/json"
+            },
+            "schemaHeaders": {
+              "Magento-Store-View-Code": "default",
+              "Magento-Website-Code": "base",
+              "Magento-Store-Code": "main_website_store",
+              "Magento-Environment-Id": "<your_environment_id>",
+              "x-api-key": "search_gql",
+              "Content-Type": "application/json"
+            }
+          }
+        }
+      }
+    ]
+  },
+  "meshId": "continuing-aqua-bobby-2pUkDmZd",
+  "lastUpdatedBy": {
+    "firstName": "User",
+    "lastName": "Name",
+    "userEmail": "uname@domain.com",
+    "userId": "undefined",
```

Comment:
Since we are returning the users' adobeID here, it would be better to have some form of alphanumeric string here instead of "undefined". 
` "userId": "A4BF2F3C61FC531A0A494210@AdobeID",`
Can be a dummy string there.